specification/compute/resource-manager/Microsoft.Compute/stable/2022-03-01/ComputeRP/examples/dedicatedHostExamples/DedicatedHostGroup_CreateOrUpdate.json

```diff
@@ -14,7 +14,10 @@
       ],
       "properties": {
         "platformFaultDomainCount": 3,
-        "supportAutomaticPlacement": true
+        "supportAutomaticPlacement": true,
+        "additionalCapabilities": {
+          "ultraSSDEnabled": true
```

Comment:
we should add a separate example file for UltraSSD and not add in the existing basic ADH creation example file.